manifests/plugin/salt.pp

```diff
@@ -52,12 +52,7 @@
     enabled   => $enabled,
     listen_on => $listen_on,
   }
-
-  file { "${foreman_proxy::etc}/salt/master.d":
```

Comment:
I think you should continue to manage the directory. Otherwise file creation can fail.

Reply:
Ok, so the file is owned by salt-master:
```console
# rpm -qf /etc/salt/master.d/
salt-master-3006.6-0.x86_64
```